Multisig/UI/App/UIKit/Transaction/TransactionDetailsViewController/Transaction Detail Cells/ExternalURLCell.swift

```diff
@@ -22,14 +22,6 @@ class ExternalURLCell: UITableViewCell {
     }
     
     @IBAction func didTapButton(_ sender: Any) {
-        UIApplication.shared.sendAction(#selector(UIViewController.didTapExternalURLCell(_:)), to: nil, from: self, for: nil)
-    }
-}
-
-extension UIViewController {
-    @objc func didTapExternalURLCell(_ sender: ExternalURLCell) {
-        if let url = sender.url {
-            openInSafari(url)
-        }
+        UIApplication.shared.sendAction(#selector(UIViewController.didTapExternalURL(_:)), to: nil, from: self, for: nil)
```

Comment:
even more cool, we can now have an extension on the "ExternalURLSource" and wrap this call in a method, so that we can just call "openExternalURL()"